ci.py

```diff
@@ -42,9 +42,12 @@
 METRIC_PIPELINE_MODULE_TO_COMPOSE_SERVICES = {
     'monasca-agent-forwarder': 'agent-forwarder',
     'zookeeper': 'zookeeper',
+    'zookeeper-watcher': 'zookeeper',
```

Comment:
should this be `'sookeeper-watcher': 'zookeeper-watcher',` ?

Reply:
yes, good catch